requirements.txt

```diff
@@ -62,3 +62,7 @@ marshmallow==2.15.3
 # License: Apache 2.0
 # Upstream url: https://pypi.org/project/responses/
 responses==0.9.0
+
+amundsen-common==0.1.7
+marshmallow==2.15.3
```

Comment:
1. We already have `marshmallow==2.15.3` on line 59
2. Could we add similar comments like we have listed above for `marshmallow-annotations`

Reply:
Oh! Good catch!